### DISCUSSION TITLE
RetrieveAttributeRequest is not yet supported... .

### DISCUSSION THREAD

**Discussion by smanikandan23** [DynamicsValue/fake-xrm-easy#171] [General]
Exception occurred: The organization request type 'Microsoft.Xrm.Sdk.Messages.RetrieveAttributeRequest' is not yet supported... . any workaround for it?

**Comment by jordimontana82** [comment]
Sorry for the delay on this @smanikandan23 . That request is actually implemented but you need to include the Messages package and also use EntityMetadata for the RetrieveAttributeRequest.

Please check:
https://dynamicsvalue.github.io/fake-xrm-easy-docs/troubleshooting/organization_request_type_not_supported/

Then:
https://dynamicsvalue.github.io/fake-xrm-easy-docs/quickstart/advanced/metadata/
